### ISSUE TITLE
make direct advertising deals (e.g. with Free-DC)

### ISSUE THREAD

**Issue by Erkan-Yilmaz** [Erkan-Yilmaz/Gridcoin-tasks#88]
from [here](https://steemit.com/gridcoin/@vortac/gridcoin-advertising-3-assessing-the-efficiency-of-our-adwords-campaigns):
"Willy de Zutter (BOINCstats webmaster) hasn't replied to our offers for a direct advertising deal (fourth so far) and we never had a direct deal with [Free-DC](http://stats.free-dc.org/)"

**Comment by Vortac2** [comment]
OK, some success here. Willy de Zutter from BOINCstats has responded to my e-mails and he is willing to carry our banner again, but the price is now 550 euros. Quite expensive, but it could be worthwhile (it's been almost a year since our last campaign on BOINCstats).

**Comment by gustav1976** [comment]
yep, boincstats should be targeted again this year. next year we might be able to offer freedome of team requirement, but shouldn't wait for that ;)

**Comment by Vortac2** [comment]
A new direct advertising deal has been negotiated with BOINCstats. Now it's time to show some support with your voting weight :)

https://steemit.com/gridcoin/@vortac/gridcoin-advertising-5-new-campaign-on-boincstats

**Comment by Vortac2** [comment]
Apparently, direct advertising deal with BOINCstats has been rejected by the community. I have arranged another (much cheaper) direct deal with Netsoft, with another corresponding Foundation vote. 

More details here:
https://steemit.com/gridcoin/@vortac/gridcoin-advertising-6-extending-the-netsoft-deal
